src/stream/StreamPart.js

```diff
@@ -0,0 +1,22 @@
+export default class StreamPart {
+    constructor(streamId, streamPartition) {
+        this._streamId = streamId
+        this._streamPartition = streamPartition
+    }
+
+    static fromStream({ id, partitions }) {
+        const result = []
+        for (let i = 0; i < partitions; i++) {
+            result.push(new StreamPart(id, i))
+        }
+        return result
+    }
```

Comment:
This is weirdly named, seems like it should be plural at least. `fromX` sounds like it should create a `StreamPart` instance from X, but this gives an array of `StreamPart` instances.

`getStreamPartsFromStream` is a bit long but maybe clearer?